routes/bots.js

```diff
@@ -152,6 +152,15 @@ botsRouter.route('/addItem').post(async (req, res) => {
 		bot.inventory.forEach((article) => {
 			if (article.item == itemId) {
 				isNewInventoryItem = false;
+				// if quantity is less than 0, it indicates the item has been bought, so add it to item's sale statistics
+				if (parseInt(quantity) < 0)
+					article.set({
+						sales: {
+							numSold:
+								parseInt(article.sales.numSold) + Math.abs(parseInt(quantity)),
+						},
+					});
+
```

Comment:
Maybe let's have another endpoint for this? It's not really clear that an endpoint called "addItem" will be used to signal an item being sold.